tests/cs/DojoSamples.Tests/DojoSamples.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>

```

Comment:
Could we use the net6.0 instead?

Reply:
Sure. I think my default is 5 but shouldn't be a problem to update.